cypress/e2e/pages/dualIndexPlate.cy.ts

```diff
@@ -230,16 +230,17 @@ describe('Dual Index Plate', () => {
           );
         });
         cy.url().reload();
+        selectSGPNumber('SGP1008');
         scanInSourceLabware('300051128832186720221202');
         scanInDestinationLabware();
+        selectOption('plateType', 'Fresh frozen - Dual Index TT Set A');
         cy.get('#sourceLabwares').within(() => {
           cy.findByText('A1').click();
         });
         cy.get('#destLabwares').within(() => {
           cy.findByText('A1').click();
         });
-        selectSGPNumber('SGP1008');
-        selectOption('plateType', 'Fresh frozen - Dual Index TT Set A');
+        cy.screenshot();
```

Comment:
Why has this `screenshot` been added?

Reply:
The Cypress tests were failing on CI at that step, and I'm not sure why. I added the screenshot to check the reason, but then the CI passed. The instruction needs to be removed, though.